### PR TITLE
adding a new api to send push notifications to all the active mobile apps

### PR DESCRIPTION
**Pull request by ihtram**

### CODE REVIEW

**Comment by ihtram** [comment]
@ziafazal can you please review this?

File: mobileapps/views.py [hunk 1]
```diff
@@ -388,6 +387,57 @@ def delete(self, request, mobile_app_id):
...
+        message = request.data.get('message', None)
+        if not message:
+            return Response({'message': _('message is missing')}, status.HTTP_400_BAD_REQUEST)
+        payload = {
+            'title': message,
+            'send_to_all': True
+        }
+
+        try:
+            mobile_apps = MobileApp.objects.filter(is_active=True)
```
Comment (ziafazal) [inline mobileapps/views.py]: nit: what if we add `notification_provider__isnull=False` filter here to avoid the `if` statement below?.

**Review by ziafazal — APPROVED** [review]
@ihtram one nit other than that LGTM.

File: mobileapps/views.py (hunk 1 above)
Comment (ihtram) [reply mobileapps/views.py]: makes sense. let me implement this one

**Comment by ihtram** [comment]
@ziafazal  your suggestions implemented. please take a look

**Pull request merged**